src/wdl_aid/wdl_aid.py

```diff
@@ -0,0 +1,303 @@
+                    "the parameter_meta sections.")
+    parser.add_argument("wdlfile", type=str,
+                        help="The WDL the documentation should be generated "
+                             "for.")
+    parser.add_argument("-o", "--output", type=Path,
+                        help="The file to write the generated documentation "
+                             "to.")
+    parser.add_argument("-t", "--template", type=Path,
+                        help="A jinja2 template to use for rendering the "
+                             "documentation. A default template will be "
+                             "used to generate a markdown file if not "
+                             "specified.")
+    parser.add_argument("-c", "--category-key", type=str, default="category",
+                        help="The key used in the parameter_meta sections "
+                             "for the input category. [category]")
+    parser.add_argument("-d", "--description-key", type=str,
+                        default="description",
+                        help="The key used in the parameter_meta section for "
+                             "the input description. [description]")
+    parser.add_argument("--do-not-separate-required", action="store_true",
+                        help="Do not put required inputs into a separate "
+                             "'required' category, but keep them in the. "
+                             "category as noted in the parameter_meta "
+                             "sections.")
+    parser.add_argument("--fallback-description-to-object",
+                        action="store_true",
+                        help="Use the entire parameter_meta object as "
+                             "description if the description key is not "
+                             "found.")
+    parser.add_argument("--fallback-description", type=str, default="???",
+                        help="The fallback value for when no description is "
+                             "defined for a given input. [???]")
+    parser.add_argument("--fallback-category", type=str, default="other",
+                        help="The fallback value for when no category is "
+                             "defined for a given input. [other]")
+    return parser.parse_args()
+
+
+def main():
+    args = parse_args()
+    doc = WDL.load(args.wdlfile)
+
+    wf = doc.workflow
+    inputs = fully_qualified_inputs(wf.available_inputs, wf.name)
+    inputs = sorted(inputs, key=lambda i: i[0])
+
+    if args.do_not_separate_required:
+        required_inputs = []
+    else:
+        required_inputs = fully_qualified_inputs(wf.required_inputs, wf.name)
+        required_inputs = [required_input[0]
+                           for required_input in required_inputs]
+
+    parameter_meta = gather_parameter_meta(wf, wf.name)
+
+    values = {"workflow_name": wf.name, "wdl_aid_version": "0.1-experimental"}
```

Comment:
Hardcoded version is not so nice. Only superhumans will never forget to update this. 
There are ways around this. But the cleanest way in my opinion is to get it from the installed package using pkg_resources. (Warning: This is a very very opinionated subject, there is no consensus whatsoever on how to deal with versions in python code). A lot of people prefer a `__version__` variable in `__init.py__`. That is sort of standard.
You could do that, and populate that version using pkg_recourses, such as in this example: 
```python
import pkg_resources

package = pkg_resources.get_distribution("wdl_aid")
# The short X.Y version
version = package.parsed_version.base_version
# The full version, including alpha/beta/rc tags
release = package.version

Reply:
:+1: